bloggies_backend/src/models/stripe.ts

```diff
@@ -0,0 +1,37 @@
+import Stripe from "stripe";
+require("dotenv").config({ path: require("find-config")(".env") });
```

Comment:
For importing `.env` values, it might be better to keep them in the `config.ts` file and export it from there to use in here.

Reply:
If possible, try keeping the import syntax to be ES6 for uniformity

Reply:
went ahead and fixed this in the commits below, let me know if there's anything else!